api_app/visualizers_manager/visualizers/passive_dns/visualizer.py

```diff
@@ -0,0 +1,117 @@
+from logging import getLogger
+from typing import Dict, List
+
+from api_app.visualizers_manager.classes import VisualizableObject, Visualizer
+from api_app.visualizers_manager.decorators import (
+    visualizable_error_handler_with_params,
+)
+from api_app.visualizers_manager.enums import VisualizableTableColumnSize
+from api_app.visualizers_manager.visualizers.passive_dns.analyzer_extractor import (
+    extract_circlpdns_reports,
+    extract_dnsdb_reports,
+    extract_mnemonicpdns_reports,
+    extract_otxquery_reports,
+    extract_robtex_reports,
+    extract_threatminer_reports,
+    extract_validin_reports,
+)
+from api_app.visualizers_manager.visualizers.passive_dns.visualize_report import (
+    visualize_report,
+)
+
+logger = getLogger(__name__)
+
+
+class PassiveDNS(Visualizer):
+    @classmethod
+    def update(cls) -> bool:
+        pass
+
+    @visualizable_error_handler_with_params("pdns_table")
+    def _pdns_table_ui(self, reports) -> VisualizableObject:
```

Comment:
i think we can move the columns definition and the return of the table in the function visualize_report. I think we won't need to customize the table for different visualizers.